### PR TITLE
Switch from Kinetic to Melodic in docs

### PR DESCRIPTION
**Pull request by davetcoleman**
closes https://github.com/ros-planning/moveit_tutorials/issues/230

### CODE REVIEW

File: doc/benchmarking/benchmarking_tutorial.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
 Benchmarking
 =====================
 
-.. note:: This is the new benchmarking method only available in ROS Kinetic, onward.
+.. note:: This is the new benchmarking method only available in ROS Melodic, onward.
```
Comment (mlautman) [inline doc/benchmarking/benchmarking_tutorial.rst]: This seems incorrect

File: doc/stomp_planner/stomp_planner_tutorial.rst [hunk 2]
```diff
@@ -122,13 +122,13 @@ STOMP has some parameters associated with it. These can be modified for the give
...
+- *class*: here you can set the cost function you want to use. You could set this to "CollisionCheck", "ObstacleDistanceGradient" or "ToolGoalPose". Depending on what you put here, you need to set the appropriate cost function class's parameters: For "CollisionCheck", you need to set the parameters (collision_penalty, cost_weight, kernel_window_percentage, longest_valid_joint_nove); for "ObstacleDistanceGradient", you should set the parameters (cost_weight, max_distance, longest_valid_joint_move) and for "ToolGoalPose", you should set the parameters (constrained_dofs, position_error_range, orientation_error_range, position_cost_weight, orientation_cost_weight). Have a look at `this link <https://github.com/ros-industrial/industrial_moveit/blob/melodic-devel/stomp_plugins/example_pages.dox>`_ for setting parameters for "ToolGoalPose" class.
```
Comment (mlautman) [inline doc/stomp_planner/stomp_planner_tutorial.rst]: broken link

File: README.md [hunk 3]
```diff
@@ -8,17 +8,15 @@ These tutorials use the [reStructuredText](http://www.sphinx-doc.org/en/stable/r
...
-
-
-
 ## Versions
 
-- ``indigo-devel`` branch should be considered for the most part "frozen" for historical reasons
-- ``kinetic-devel`` branch is latest
-- ``melodic-devel`` branch is coming soon, see [issue](https://github.com/ros-planning/moveit_tutorials/issues/230)
+- ``indigo-devel``
+- ``kinetic-devel``
```
Comment (mlautman) [inline README.md]: Note that this is the stable version and Melodic is still in Beta

File: doc/benchmarking/benchmarking_tutorial.rst (hunk 1 above)
Comment (davetcoleman) [reply doc/benchmarking/benchmarking_tutorial.rst]:
```suggestion
.. note:: This is the new benchmarking method only available in ROS Kinetic, onward.
```

Comment (davetcoleman) [reply doc/benchmarking/benchmarking_tutorial.rst]: use the suggest feature going forward!

File: README.md (hunk 3 above)
Comment (davetcoleman) [reply README.md]: i'd shy away from calling this beta as melodic was released last May

Comment (davetcoleman) [reply README.md]: technically all of moveit is in beta ;-)

i've added that kinetic "is stable" to imply that melodic is not. melodic now says "latest, changes should target this branch"

File: doc/chomp_planner/chomp_planner_tutorial.rst [hunk 4]
```diff
@@ -14,7 +14,7 @@ You should also have gone through the steps in `Visualization with MoveIt! RViz
 
 Prerequisites
 --------------
- 1. You must have the latest version of MoveIt! installed. On ROS Kinetic you will need to build MoveIt! from source. A build from source is required as CHOMP is not part of the official release yet. It is therefore not included in the binary packages. We will go through the steps for doing this below.
+ 1. You must have the latest version of MoveIt! installed. On ROS Melodic you will need to build MoveIt! from source. A build from source is required as CHOMP is not part of the official release yet. It is therefore not included in the binary packages. We will go through the steps for doing this below.
```
[the remark this answers is not in the document]
Comment (davetcoleman) [reply doc/chomp_planner/chomp_planner_tutorial.rst]: Clearly I just did a find-replace here :)

I've reworded the instructions to say you don't need to install from source anymore for Melodic

File: doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst [hunk 5]
```diff
@@ -145,7 +145,7 @@ Step 4: Use Motion Planning with the Panda
 
 Introspecting Trajectory Waypoints
 ++++++++++++++++++++++++++++++++++
-*Available since: Indigo 0.7.10, Kinetic and Lunar 0.9.7*
+*Available since: Indigo 0.7.10, Melodic and Lunar 0.9.7*
```
[the remark this answers is not in the document]
Comment (davetcoleman) [reply doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst]: Removed entirely

File: doc/setup_assistant/setup_assistant_tutorial.rst [hunk 6]
```diff
@@ -23,9 +23,9 @@ MoveIt! and ROS
   first if you have not already done that.
 
 * If you haven't already done so, make sure you have the `Franka description
-  package <https://github.com/frankaemika/franka_ros>`_ for Kinetic: ::
+  package <https://github.com/frankaemika/franka_ros>`_ for Melodic: ::
 
-   sudo apt-get install ros-kinetic-franka-description
+   sudo apt-get install ros-melodic-franka-description
```
[the remark this answers is not in the document]
Comment (davetcoleman) [reply doc/setup_assistant/setup_assistant_tutorial.rst]: noted

**Pull request closed without merging**